### PR TITLE
[Docs] Document of RReLU about its different behavior between training and evaluation

### LINKED ISSUE #95605: The doc of `torch.nn.RReLU` does not demonstrate its difference between training and eval
### 📚 The doc issue

I'm reading the doc of [Randomized Leaky ReLU](https://pytorch.org/docs/stable/generated/torch.nn.RReLU.html#torch.nn.RReLU). As we know, its behavior is different in training mode and eval mode. In training mode `alpha` is sampled from an uniform distribution while in evaluation mode `alpha = (lower + upper) / 2`. I think the documentation can add some illustrations about this.

### Suggest a potential alternative/fix

The original doc in RReLU:
```
where :math:`a` is randomly sampled from uniform distribution
    :math:`\mathcal{U}(\text{lower}, \text{upper})`.
```
Can be modified as:
```
where :math:`a` is randomly sampled from uniform distribution
    :math:`\mathcal{U}(\text{lower}, \text{upper})` during training while during 
evaluation :math:`a` is fixed with :math:`a = \frac{\text{lower} + \text{upper}}{2}`.
```

cc @svekars @carljparker

### PR DESCRIPTION
**Pull request by SiriusNEO**
Current document of [Randomized Leaky ReLU (RReLU)](https://pytorch.org/docs/stable/generated/torch.nn.RReLU.html#torch.nn.RReLU) does not demonstrate its different behavior between training and evaluation. This PR adds illustrations about this.

Fixes #95605.

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Thanks for the update!

**Comment by albanD** [comment]
Ho could you remove the trailing space to fix the lint please?

**Comment by SiriusNEO** [comment]
@albanD Sorry, fixed now

**Comment by SiriusNEO** [comment]
@albanD @H-Huang Hey guys, could you help merge this PR? Thanks!

**Review by H-Huang — APPROVED** [review]
LGTM!
